tiatoolbox/utils/misc.py

```diff
@@ -103,7 +103,8 @@ def imwrite(image_path, img):
     """Write a numpy array to an image
 
     Args:
-        image_path (str, pathlib.Path): file path (including extension) to save image
+        image_path (str, pathlib.Path): file path (including extension)
+        to save image
```

Comment:
Maybe needs indenting

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/25809e3ee8955f97933d290bd7d1b0874f7c6b15